app/assets/javascripts/tooltip.js

```diff
@@ -0,0 +1,19 @@
+var showTooltips = function(options) {
```

Comment:
@houndci is correct with this - there is no need to define this.

_Tip_: when wrapping you whole script in an anonymous function, you can reduce the amount of "`jQuery`" you can write:

```javascript
!(function($) {
  // $ is jQuery - this will not collide with the $ for  prototype.js
  $.extend( ... ) // your snippets
}(jQuery))
```